crates/core/src/library.rs

```diff
@@ -108,6 +108,25 @@ impl Library {
     ///     move |s: String| s.len() * factor
     /// }
     /// ```
+    ///
+    /// When the `bevy` feature is set it is possible to register bevy systems as functions.
```

Comment:
Can we reorder this so that the Bevy use case comes first? We can then have a section like "Usage without Bevy" that shows the old function syntax.